api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml

```diff
@@ -117,4 +117,15 @@
 								 baseTableName="patient_state" baseColumnNames="encounter_id"
 								 referencedTableName="encounter" referencedColumnNames="encounter_id" />
 	</changeSet>
+
+	<changeSet id="2021-09-02-TRUNK-6020" author="tendomart">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="user_property" columnName="property"></columnExists>
+			<columnExists tableName="user_property" columnName="property_value"></columnExists>
```

Comment:
These should be two separate changesets.

Reply:
Alright let me have it fixed.